iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -682,7 +740,7 @@ public static Schema schema(Configuration config) {
   }
 
   /**
-   * Stores the serializable table data in the configuration.
+   * Stores the serializable table data in the configuration
```

Comment:
Why is this change?
BTW Javadoc expect `.` at the end of the first sentence... at least I remember somewhere fighting with adding extra dots at the end of the lines 😄